api/app/Policies/PoolPolicy.php

```diff
@@ -174,12 +166,13 @@ public function closePool(User $user, Pool $pool)
      */
     public function deleteDraft(User $user, Pool $pool)
     {
-        $pool->loadMissing('team');
-        if ($pool->getStatusAttribute() !== PoolStatus::DRAFT->name) {
-            return Response::deny("You cannot delete a Pool once it's been published.");
+        if ($pool->getStatusAttribute() === PoolStatus::DRAFT->name) {
+            $pool->loadMissing('team');
+
+            return $user->isAbleTo('delete-team-draftPool', $pool->team);
         }
 
-        return $user->isAbleTo('delete-team-draftPool', $pool->team);
+        return false;
```

Comment:
I miss the `Response::deny` here too

Reply:
I've (re)added some error responses with [Update pool delete policy with specific denial messages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8312/commits/97cebccf478afceb30d5c4e86c40f75597bc8265).